src/server/game/AI/CoreAI/UnitAI.cpp

```diff
@@ -291,6 +291,16 @@ void UnitAI::FillAISpellInfo()
     }
 }
 
+ThreatMgr& UnitAI::GetThreatMgr()
+{
+    return me->GetThreatMgr();
+}
+
+void UnitAI::SortByDistance(std::list<Unit*> list, bool ascending)
```

Comment:
This list is copied and then sorted. Is that intended? Perhaps the `list` argument should be a reference.